src/System.Net.Http/tests/FunctionalTests/DiagnosticsTests.cs

```diff
@@ -468,11 +468,18 @@ public void SendAsync_ExpectedDiagnosticExceptionActivityLogging()
             }, UseSocketsHttpHandler.ToString()).Dispose();
         }
 
-        [ActiveIssue(29482)]
         [OuterLoop] // TODO: Issue #11345
         [Fact]
         public void SendAsync_ExpectedDiagnosticSynchronousExceptionActivityLogging()
```

Comment:
I think it is worth renaming this to be a little more clear. Not going to block the pr on this though, as the current name seems standard for the file.

Reply:
The naming in this test file is not aligned with corefx standard. We can fix in a separate PR.